### PR TITLE
Add some selenium tests, much is pending.

### PR DESCRIPTION
**Pull request by christian-marie**
Happybarra is a misnomer, no happybaras are involved.

Happy capybara

### CODE REVIEW

File: test/use-cases.hs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            putStrLn ""
+            putStrLn "This test relies on ./runit.sh, ./proxy.sh, and selenium being run"
+            putStrLn "To set up a selenium standalone server, download it here:"
+            putStrLn "http://docs.seleniumhq.org/download/"
+            putStrLn "Then run: java -jar selenium-server-standalone*.jar"
+
+spec :: URI -> Spec
+spec uri =
+    describe "As a user," $ do
+        session "I can create a token with a subset of my scope" $ using [Firefox, Chrome] $ do
```
Comment (tvh) [inline test/use-cases.hs]: webdriver-hspec is not so great here. It doesn't expose htmlunit, which I think is the way to go for travis.

Comment (tvh) [inline test/use-cases.hs]: There is a way, defining custum instances for `TestCapabilities` and `Using`. An easy way would be to just use the `Capabilities` directly.

**Comment by thsutton** [comment]
Let us rumble!

**Comment by christian-marie** [comment]
lettuce

**Pull request merged**